FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -30,7 +30,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !UserDefaults.standard.bool(forKey: UserDefaultsKeys.hasShownWelcomeScreen.rawValue) {
             showWelcome()
         }
-        
+
+        var types = UIUserNotificationType()
+        types.insert(.alert)
+        types.insert(.badge)
+        UIApplication.shared.registerUserNotificationSettings(UIUserNotificationSettings(types: types, categories: nil))
```

Comment:
`UIUserNotification*` is an iOS 10-only framework. We'll have to include back-ported support for iOS 9 devices.

According to this chart:

![image](https://cloud.githubusercontent.com/assets/59140/23070837/ffaef474-f4f1-11e6-9a83-bd0d323b5e64.png)

We have roughly3% of our devices that run iOS 9. (these numbers are just those that opt-in to iTunes Analytics)

Maybe a case could be made to make this iOS 10 only, but for now I think it's easy enough to keep it iOS 9 compatible.